jazzhands/__main__.py

```diff
@@ -175,16 +188,14 @@ def main(argv):
             js_dir = root
         
         for fn in files:
-            if fn.endswith('.styl'):
-                fn = os.path.join(root, fn)
-                stylus_watch[fn] = os.stat(fn).st_mtime
-            if fn.endswith('.js'):
-                fn = os.path.join(root, fn)
-                js_watch[fn] = os.stat(fn).st_mtime
-
-    if index_files.get('styl') and index_files.get('less'):
-        print("ERROR: I don't know how to combine Stylus and Less in a single build... yet!")
-        return
+            for ext in ('js', 'styl', 'less'):
+                if fn.endswith('.' + ext):
+                    fn = os.path.join(root, fn)
+                    watch[ext][fn] = os.stat(fn).st_mtime
```

Comment:
might as well break after this line, the filename can only end with one of the extensions.

Reply:
...quantum computing super-state filenames?